SynthIPTemplate/config_DECOMPRESS_fastmode.info

```diff
@@ -0,0 +1,12 @@
+description: "Patch for configuration (DECOMPRESS, fastmode)"
```

Comment:
I would consider configurations that combine `DECOMPRESS` with `fastmode` and `smallmode` as invalid configurations since they have no effect in `DECOMPRESS` mode.
The feature model for the yalz77 based case studies should look like this:
```
         yalz77
         /XOR\
        /     \
DECOMPRESS    COMPRESS
                /OR\
               /    \
            fast    small
```

Reply:
I see... That would have been useful information somewhere :laughing: